packages/rest/src/parser.ts

```diff
@@ -65,13 +76,15 @@ async function loadRequestBodyIfNeeded(
   if (!operationSpec.requestBody) return Promise.resolve();
 
   const contentType = getContentType(request);
+  debug('Loading request body with content type %j', contentType);
   if (contentType && !/json/.test(contentType)) {
     throw new HttpErrors.UnsupportedMediaType(
       `Content-type ${contentType} is not supported.`,
     );
   }
 
   return await parseJsonBody(request).catch((err: HttpError) => {
+    debug('Cannot parse request body', err);
```

Comment:
I intentionally didn't use `%j` because I want the error to be formatted using `util.inspect`, not `JSON.stringify`.

I think we should use `%s` and `inspect(err)` as we are already doing in other places?

Reply:
I will change it in https://github.com/strongloop/loopback-next/pull/1495